modules/dxp/apps/search-experiences/search-experiences-web/src/main/resources/META-INF/resources/sxp_blueprint_admin/js/edit_sxp_blueprint/EditSXPBlueprintForm.js

```diff
@@ -232,21 +231,23 @@ function EditSXPBlueprintForm({
 					method: 'PUT',
 				}
 			).then((response) => {
-				if (!response.ok) {
-					setShowSubmitWarningModal(false);
-
-					throw DEFAULT_ERROR;
-				}
-
-				return response.json();
+				return response.json().then((data) => ({
+					ok: response.ok,
+					responseContent: data,
+				}));
 			});
 
-			if (
-				Object.prototype.hasOwnProperty.call(responseContent, 'errors')
-			) {
-				responseContent.errors.forEach((message) =>
-					openErrorToast({message})
-				);
+			if (!ok) {
+				if (Object.hasOwn(responseContent, 'title')) {
+					openErrorToast({
+						message: responseContent.title,
+					});
+				}
+				else if (Object.hasOwn(responseContent, 'errors')) {
+					responseContent.errors.forEach((message) =>
+						openErrorToast({message})
+					);
+				}
```

Comment:
@oliv-yu We should also add the default error to appear if none of these properties are found.

```js
else {
    throw DEFAULT_ERROR;
}
```

Reply:
@thektan Oops, thanks for catching that!! x_x

@gustavolimav could you add in the last commit on https://github.com/oliv-yu/liferay-portal/commits/LPS-202379-error-handling? Thanks!

Reply:
@oliv-yu I will take a look at this today